ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/MaximumLagFilterTest.java

```diff
@@ -94,7 +94,7 @@ public void filter_hostNoLag() {
         PARTITION).get();
 
     // Then:
-    assertTrue(filter.filter(HOST1));
+    assertFalse(filter.filter(HOST1));
```

Comment:
should the test name change given the semantics are inverted now? might be good to rename in the same `shouldXXX` form

Reply:
Done